### PR TITLE
Business Events database and status changes captured

### PR DESCRIPTION
**Pull request by AnaisHebert**

### CODE REVIEW

File: prime-dotnet-webapi/Models/BusinessEvent.cs [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+using System.ComponentModel.DataAnnotations.Schema;
+using System.Text.Json.Serialization;
+
+namespace Prime.Models
+{
+    [Table("BusinessEvent")]
+    public class BusinessEvent : BaseAuditable
+    {
+        [Key]
+        public int? Id { get; set; }
```
Comment (james-hollinger) [inline prime-dotnet-webapi/Models/BusinessEvent.cs]: Don't think we should have nullable indexes any more

Comment (AnaisHebert) [reply prime-dotnet-webapi/Models/BusinessEvent.cs]: oh yeah good catch

File: prime-dotnet-webapi/Models/BusinessEvent.cs [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+        public int EnrolleeId { get; set; }
+
+        [JsonIgnore]
+        public Enrollee Enrollee { get; set; }
+
+        public int? AdminId { get; set; }
```
Comment (james-hollinger) [inline prime-dotnet-webapi/Models/BusinessEvent.cs]: Admin Id is not nullable

Comment (james-hollinger) [reply prime-dotnet-webapi/Models/BusinessEvent.cs]: Wait you're right, not all events have Admin Ids

**Pull request merged**